### DISCUSSION TITLE
Welcome to StartupToolchain Discussions!

### DISCUSSION THREAD

**Discussion by abishekmuthian** [abishekmuthian/startuptoolchain#1] [Announcements]
## 👋 Welcome!
  Use the discussions to suggest new categories, tools for the StartupToolchain or for queries on existing categories and tools:  
  * Suggest new categories explaining why it needs to be on StartupToolchain.
  * Suggest new sub-categories explaining why it needs to be on StartupToolchain.
  * Suggest new tools explaining why it needs to be on StartupToolchain.

**Comment by MedusaNick** [comment]
Hi Abishek,

Thanks for creating this awesome curated list! Wanted to suggest a tool for the toolchain:

- **Name:** Medusa
- **Category:** E-commerce
- **Tag:** Open Source
- **Webpage:** https://www.medusajs.com/
- **GitHub:** https://github.com/medusajs/medusa
- **Why add it:** Because it is a completely free tool (100% OS) for people to use to build an ecommerce platform and it has in record time gained more traction w. ~10,000 GH stars after only 7 months live

Thought it might be worth adding. Once again, thanks for putting together this link. Really cool work!

**Comment by abishekmuthian** [comment]
Hi Nick,

Thanks for the message.

There's a medusa-commerce currently under [E-commerce](https://startuptoolchain.com/#e-commerce) but the website seems to be down, Have you rebranded or is your project not related to medusa-commerce?

Edit: Medusa-Commerce has been updated to medusa, URL changed to the medusajs.com and alpha superscript has been removed.

**Reply by MedusaNick** [reply]
That is perfect, thanks for the help!

One more thing, can we add the OS-tag to Medusa? Since the solution is completely open source :-)
